### PR TITLE
#166325993 Fixing bug when upload image

### PR DESCRIPTION
**Pull request by djallas**
#### What does this PR do?

This PR fixes the bug occurs when uploading images
- Save the image in the database so that a user can access his gallery
- Remove coverUrl as required when creating a new article
- Fetch user's gallery

#### What are the relevant pivotal tracker stories?
#166325993

### CODE REVIEW

**Comment by djallas** [comment]
@e-liyai you can check this.

File: src/controllers/ArticleController.js [hunk 1]
```diff
@@ -11,15 +11,14 @@ export default class ArticleController {
...
-    const image = await helpers.upload(req);
     const { tagList } = req.body;
     const newArticle = await Article.create({
       userId: req.user.id || 0,
       slug: helpers.generator.slug(req.body.title),
       title: req.body.title.trim(),
       description: req.body.description.trim(),
       body: req.body.body.trim(),
-      coverUrl: image.image.original,
+      coverUrl: req.body.coverUrl || '/templates/images/image.png',
```
Comment (e-liyai) [inline src/controllers/ArticleController.js]: is the default `coverURL` value supposed to help with those that don't want to upload images?

Comment (djallas) [reply src/controllers/ArticleController.js]: YES

Comment (e-liyai) [reply src/controllers/ArticleController.js]: i feel like there should be a more elaborate way of doing this, we don't have to add a dummy image

**Comment by djallas** [comment]
@e-liyai PR Updated

File: src/tests/controllers/ArticleController.test.js [hunk 2]
```diff
@@ -177,6 +174,44 @@ describe('Article', () => {
...
+        expect(res.body.errors.coverUrl).to.equal('coverUrl not updated, try again');
+      });
+  });
+  it('Should get gallery', async () => {
+    chai
+      .request(app)
+      .get('/api/v1/gallery')
+      .set('access-token', accessToken)
+      .end((err, res) => {
+        console.log('rw', res.body);
```
Comment (e-liyai) [inline src/tests/controllers/ArticleController.test.js]: remove console log and commented out code.

Comment (djallas) [reply src/tests/controllers/ArticleController.test.js]: @e-liyai solved

**Pull request merged**